wandb/data_types.py

```diff
@@ -712,7 +712,7 @@ def to_json(self, run_or_artifact):
 
         elif isinstance(run_or_artifact, wandb_artifacts.Artifact):
             for column in self.columns:
-                if "." in column:
+                if isinstance(column, str) and "." in column:
```

Comment:
Dont know where this string came from, but for python2 you might need;
```
if isinstance(value, six.string_types):
```

Reply:
ahh, it is possible for column to be an int, so the 'in' operator is no good... ill fix this now